plugins/Delay/DelayControlsDialog.cpp

```diff
@@ -22,10 +22,13 @@
  *
  */
 
-#include "delaycontrolsdialog.h"
-#include "delaycontrols.h"
+#include "DelayControlsDialog.h"
+#include "DelayControls.h"
 #include "embed.h"
 #include "TempoSyncKnob.h"
+#include "../Eq/EqFader.h"
```

Comment:
Hmm.... Relying on another plugin might be taboo here... Perhaps we should bring this `EqFader` class back into our core code?